scripts - Restore packages.bat

```diff
@@ -1,4 +0,0 @@
-for /f %%a in ('dir /b src\*.sln') do call tools\nuget\nuget.exe restore src\%%a -PackagesDirectory .\lib
```

Comment:
The idea is to restore with just a double click. If you prefer the "nuget restore" thing, that's great, but I believe we don't have to remove this in order to make nuget restore work.